spec/requests/v2/ont/runs_spec.rb

```diff
@@ -1,27 +1,80 @@
 # frozen_string_literal: true
 
-require "rails_helper"
+require 'rails_helper'
 
 RSpec.describe 'GraphQL', type: :request do
-  let(:run_factory) { instance_double("Ont::RunFactory") }
+  context 'get run' do
+    context 'when there is a valid run' do
```

Comment:
Does it make sense to also test 'when there is not a valid run' since you've split these into the opposite context?